crypto/bn/bn_dh.c

```diff
@@ -204,8 +485,12 @@ static const BN_ULONG dh2048_256_q[] = {
                        const BIGNUM _bignum_##x = { (BN_ULONG *) x, \
                         OSSL_NELEM(x),\
                         OSSL_NELEM(x),\
-                        0, BN_FLG_STATIC_DATA };
+                        0, BN_FLG_RDONLY };
```

Comment:
What's rationale behind adding RDONLY? Why STATIC_DATA isn't sufficient?

Reply:
Have a look at BN_free and BN_clear_free, both can modify the BIGNUM and there could be compatibility issues if that is changed. So added a new flag BN_FLG_RDONLY to indicate it shouldn't be touched at all.